apps/bfd-model/bfd-model-rif-samples/src/main/resources/rif-static-samples/sample-u-bcarrier.txt

```diff
@@ -1,2 +1,2 @@
 DML_IND|BENE_ID|CLM_ID|CLM_GRP_ID|FINAL_ACTION|NCH_NEAR_LINE_REC_IDENT_CD|NCH_CLM_TYPE_CD|CLM_FROM_DT|CLM_THRU_DT|NCH_WKLY_PROC_DT|CARR_CLM_ENTRY_CD|CLM_DISP_CD|CARR_NUM|CARR_CLM_PMT_DNL_CD|CLM_PMT_AMT|CARR_CLM_PRMRY_PYR_PD_AMT|RFR_PHYSN_UPIN|RFR_PHYSN_NPI|CARR_CLM_PRVDR_ASGNMT_IND_SW|NCH_CLM_PRVDR_PMT_AMT|NCH_CLM_BENE_PMT_AMT|NCH_CARR_CLM_SBMTD_CHRG_AMT|NCH_CARR_CLM_ALOWD_AMT|CARR_CLM_CASH_DDCTBL_APLD_AMT|CARR_CLM_HCPCS_YR_CD|CARR_CLM_RFRNG_PIN_NUM|PRNCPAL_DGNS_CD|PRNCPAL_DGNS_VRSN_CD|ICD_DGNS_CD1|ICD_DGNS_VRSN_CD1|ICD_DGNS_CD2|ICD_DGNS_VRSN_CD2|ICD_DGNS_CD3|ICD_DGNS_VRSN_CD3|ICD_DGNS_CD4|ICD_DGNS_VRSN_CD4|ICD_DGNS_CD5|ICD_DGNS_VRSN_CD5|ICD_DGNS_CD6|ICD_DGNS_VRSN_CD6|ICD_DGNS_CD7|ICD_DGNS_VRSN_CD7|ICD_DGNS_CD8|ICD_DGNS_VRSN_CD8|ICD_DGNS_CD9|ICD_DGNS_VRSN_CD9|ICD_DGNS_CD10|ICD_DGNS_VRSN_CD10|ICD_DGNS_CD11|ICD_DGNS_VRSN_CD11|ICD_DGNS_CD12|ICD_DGNS_VRSN_CD12|CLM_CLNCL_TRIL_NUM|CARR_CLM_CNTL_NUM|LINE_NUM|CARR_PRFRNG_PIN_NUM|PRF_PHYSN_UPIN|PRF_PHYSN_NPI|ORG_NPI_NUM|CARR_LINE_PRVDR_TYPE_CD|TAX_NUM|PRVDR_STATE_CD|PRVDR_ZIP|PRVDR_SPCLTY|PRTCPTNG_IND_CD|CARR_LINE_RDCD_PMT_PHYS_ASTN_C|LINE_SRVC_CNT|LINE_CMS_TYPE_SRVC_CD|LINE_PLACE_OF_SRVC_CD|CARR_LINE_PRCNG_LCLTY_CD|LINE_1ST_EXPNS_DT|LINE_LAST_EXPNS_DT|HCPCS_CD|HCPCS_1ST_MDFR_CD|HCPCS_2ND_MDFR_CD|BETOS_CD|LINE_NCH_PMT_AMT|LINE_BENE_PMT_AMT|LINE_PRVDR_PMT_AMT|LINE_BENE_PTB_DDCTBL_AMT|LINE_BENE_PRMRY_PYR_CD|LINE_BENE_PRMRY_PYR_PD_AMT|LINE_COINSRNC_AMT|LINE_SBMTD_CHRG_AMT|LINE_ALOWD_CHRG_AMT|LINE_PRCSG_IND_CD|LINE_PMT_80_100_CD|LINE_SERVICE_DEDUCTIBLE|CARR_LINE_MTUS_CNT|CARR_LINE_MTUS_CD|LINE_ICD_DGNS_CD|LINE_ICD_DGNS_VRSN_CD|HPSA_SCRCTY_IND_CD|CARR_LINE_RX_NUM|LINE_HCT_HGB_RSLT_NUM|LINE_HCT_HGB_TYPE_CD|LINE_NDC_CD|CARR_LINE_CLIA_LAB_NUM|CARR_LINE_ANSTHSA_UNIT_CNT
-UPDATE|567834|9991831999|900|N|O|71|27-OCT-1999|27-OCT-2000|06-NOV-1999|1|1|61026|1|199.99|0|1234534|8765676|A|123.45|0|245.04|166.23|0|5|K25852|H5555|0|H5555|0|H8888|0|H66666|0|H77777|0|||||||||||||||||0|74655592568216|6|K25555||1923124|5555553305|0|204299999|IL|555558202|41|1|0|1|1|11|15|27-OCT-1999|27-OCT-1999|92999|LT||T2D|37.5|0|37.5|0|E|0|9.57|75|47.84|A|0|0|1|3|H12345|0|||42.0|R1|777666666|GG443333HH|0|000000000
+UPDATE|567835|9991831999|900|N|O|71|27-OCT-1999|27-OCT-2000|06-NOV-1999|1|1|61026|1|199.99|0|1234534|8765676|A|123.45|0|245.04|166.23|0|5|K25852|H5555|0|H5555|0|H8888|0|H66666|0|H77777|0|||||||||||||||||0|74655592568216|6|K25555||1923124|5555553305|0|204299999|IL|555558202|41|1|0|1.0|1|11|15|27-OCT-1999|27-OCT-1999|92999|LT||T2D|37.5|0|37.5|0|E|0|9.57|75|47.84|A|0|0|1|3|H12345|0|||42.0|R1|777666666|GG443333HH|0|000000000
```

Comment:
I understand the change from 1 to 1.0 but I wasn't expecting the bene_id to change from 567834 to 567835. What was the reason for that?

Reply:
GEEZ, no idea how that even got like that...thanks for catching that! Fixing now...